components/host-dashboard/reports-section/TotalMoneyManagedSection.js

```diff
@@ -111,7 +111,7 @@ const TotalMoneyManagedSection = ({ currency, hostMetrics }) => {
             <TotalFundsLabel
               minWidth="230px"
               position="relative"
-              left={['-210px', '-210px', '-210px', '320px']}
+              left={['-210px', '-210px', '-210px', '130px']}
```

Comment:
As per https://github.com/opencollective/opencollective/issues/4690, we should solve that by using a responsive mechanism like CSS's flex, otherwise it'll just pop again whenever we'll resize the container. Let's take that out of this PR and solve it with a more future-proof solution.

Reply:
I'll open a separate PR for this soon using the flex approach. 🤔 👍🏽